From 595de95272d40f7ddb47c22c187a4bf19982de35 Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Thu, 2 Mar 2017 14:21:37 +0100
Subject: [PATCH] RHEL 7: v2v: -i disk: force VNC as display (RHBZ#1372671)

The SDL output mode is not supported in RHEL 7's qemu-kvm.
---
 v2v/input_disk.ml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/v2v/input_disk.ml b/v2v/input_disk.ml
index 7ecd19fd3..e2b16cf70 100644
--- a/v2v/input_disk.ml
+++ b/v2v/input_disk.ml
@@ -87,7 +87,7 @@ class input_disk input_format disk = object
       s_features = [ "acpi"; "apic"; "pae" ];
       s_firmware = UnknownFirmware;       (* causes virt-v2v to autodetect *)
       s_display =
-        Some { s_display_type = Window; s_keymap = None; s_password = None;
+        Some { s_display_type = VNC; s_keymap = None; s_password = None;
                s_listen = LNoListen; s_port = None };
       s_video = None;
       s_sound = None;
-- 
2.17.2

